agent/agent.go

```diff
@@ -343,15 +346,6 @@ func (a *Agent) setupLogging() error {
 	return nil
 }
 
-func (a *Agent) SetTestingMode(enabled bool) {
-	a.testingMode = enabled
-	if a.testingMode {
-		a.recorder.ChangeFlushFrequency(testingModeFrequency)
```

Comment:
We can also remove `recorder.ChangeFlushFrequency`

Reply:
removed